### PR TITLE
Create a F2010 compset that runs with P3 (F2010SC5-P3)

### PR DESCRIPTION
**Pull request by crterai**
Created a F2010 compset that runs the v2 (alpha5_59) version of the model with P3 (called F2010SC5-P3). 

* Changes cld_macmic_numstep to 6 for default CLUBB + SHOC configuration. And to 3 for ne120 simulations. 
* Sets land IC for ne30pg2_r05_oECv3 and ne120pg2_r0125_oRRS18to6v3

### CODE REVIEW

File: run_F2010SC5-P3_cori.csh [hunk 1]
```diff
@@ -0,0 +1,1519 @@
+#! /bin/csh -fe
```
Comment (singhbalwinder) [inline run_F2010SC5-P3_cori.csh:1]: Do we want to integrate this script as well or is this added by mistake?

Comment (crterai) [reply run_F2010SC5-P3_cori.csh:1]: Hi @singhbalwinder, this is just a pull request to Koby's branch that he had created for the convection assessment, so this is not for merging into master. 
Perhaps I needed to turn on a flag to indicate this?

Comment (singhbalwinder) [reply run_F2010SC5-P3_cori.csh:1]: Thanks! I didn't realize that.

**Comment by rljacob** [comment]
Don't do PR's between feature branches here on the main github repo.  One of you can move development to a fork and use PR's there.

**Pull request closed without merging**